README.md

```
@@ -144,4 +146,4 @@ Distributed under the Apache 2.0 License. See [LICENSE.md](LICENSE.md) for more
 [Electron-url]: https://www.electronjs.org/
 [Vite.js]: https://img.shields.io/badge/vite-%23646CFF.svg?logo=vite&logoColor=white
 [Vite-url]: https://vitejs.dev/
-[product-screenshot]: assets/brand.png
\ No newline at end of file
+[product-screenshot]: assets/brand.png
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "product-screenshot"

(MD053, link-image-reference-definitions)